packages/stems/src/components/HarmonyButton/HarmonyButton.module.css

```diff
@@ -0,0 +1,220 @@
+/* ===Base Styles=== */
+.button {
+  --button-color: var(--primary);
+  --text-color: var(--static-white);
+  --overlay-color: transparent;
+  --overlay-opacity: 0;
+  align-items: center;
+  border: 1px solid var(--button-color);
+  border-radius: var(--unit-1);
+  box-sizing: border-box;
+  color: var(--text-color);
+  cursor: pointer;
+  display: inline-flex;
+  flex-shrink: 0;
+  justify-content: center;
+  overflow: hidden;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  white-space: nowrap;
+}
+
+.button:focus {
+  outline: none !important;
```

Comment:
do we want to do a css reset on button? mabye `all: unset` on the root element`

Reply:
I had to look up how this works
>Specifies that all the element's properties should be changed to their inherited values if they inherit by default, or to their initial values if not.

I'm actually a bit confused on how it's useful. Could you give me an example scenario you're trying to account for?